mobile/test/objective-c/EnvoyTestServer.h

```diff
@@ -0,0 +1,21 @@
+#import <Foundation/Foundation.h>
+
+// Interface for starting and managing a test server. Calls into to test_server.cc
+@interface EnvoyTestServer : NSObject
+
+// Starts a server with HTTP3 and TLS.
++ (void)startQuicTestServer;
```

Comment:
Instead of having 3 methods which simply delegate to start_server() with different args, can we just have a single startServer() method which takes the same enum that start_server() takes (assuming we go that route).

If we can't go the enum route and need these 3 methods let's name them more consistently:
startHttp3Server()
startHttp2Server()
startHttp1PlaintextServer()
(or something similar)

Reply:
Done! Also for Java since that's where I got the names from.

Reply:
Hm I was proposing adding a single method named startServer() which took an enum as an argument. Can we do that?

Reply:
resolved offline